index.js

```diff
@@ -1,25 +1,49 @@
+/* eslint-disable no-var, prefer-template, comma-dangle */
+
 'use strict';
 
 module.exports = function buildAirbnbPreset(context, options) {
+  // This is similar to how `env` works in Babel:
+  // https://babeljs.io/docs/usage/babelrc/#env-option
+  // We are not using `env` because it’s ignored in versions > babel-core@6.10.4:
+  // https://github.com/babel/babel/issues/4539
+  // https://github.com/facebookincubator/create-react-app/issues/720
+  // It’s also nice that we can enforce `NODE_ENV` being specified.
+  var env = process.env.BABEL_ENV || process.env.NODE_ENV || 'development';
+
+  var allPlugins = [
+    [require('babel-plugin-transform-es2015-block-scoping'), { throwIfClosureRequired: true }],
```

Comment:
This is going to be a breaking change anyway, because the development plugins shouldn't be included in production, so we now require `NODE_ENV` set to production for production builds.